perun-beans/src/main/java/cz/metacentrum/perun/core/api/DBVersion.java

```diff
@@ -0,0 +1,75 @@
+package cz.metacentrum.perun.core.api;
+
+import java.util.List;
+
+/**
+ * Object for representation of database version.
+ *
+ * @author Simona Kruppova 410315
+ * @author Oliver Mrazik 410035
+ */
+public class DBVersion extends Auditable {
```

Comment:
This object not extended PerunBean and also not extended Auditable. Please remove extends part.